src/components/forms/InputGroup/InputGroup.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+import { InputGroup } from './InputGroup'
+
+describe('InputGroup component', () => {
```

Comment:
Maybe add a test for checking that the right className is applied?

Reply:
+ with the error state

Reply:
Good catch! I was just thinking the tests could probably be more thorough. Thanks!

Reply:
@jcbcapps sorry, I shoulda been more clear. Can you add a test that verifies that the component has the default class name applied? (`usa-input-group`)

It can go in the `renders without errors` test, imo, since it's jsut verifying that the component built with the right uswds class